Assets/CodePanel/CodePanelScript.cs

```diff
@@ -0,0 +1,77 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using TMPro;
+using UnityEngine.InputSystem.LowLevel;
+using UnityEngine.ProBuilder.Shapes;
```

Comment:
I would suggest removing unused things here. In our case it is not messing up anything but sometimes it can lead to errors while building a game later.